docs/news/posts/2024-12-13-core-plugins.md

```diff
@@ -0,0 +1,140 @@
+
+## Enhancing high-throughput
+
+### [`aiida-submission-controller`](https://github.com/aiidateam/aiida-submission-controller/)
+
+Have you ever wondered how a "hero run" like the one recently conducted on the new CSCS Alps infrastructure (read more about it [here](https://nccr-marvel.ch/highlights/AiiDA-hero-run-Alps)) can be orchestrated?
+In case you want to run a certain workflow, or several connected ones, for a set of structures, it can be handy to have a tool that:
+
+1. Takes care of keeping a certain number of active workflows at any time, without needing to monitor the runs.
+2. Makes sure you don't run the same structure twice.
+
+This is exactly the goal of the `aiida-submission-controller`.
+Setting up a controller simply requires that you specify how to run the process you want to run based on the parent node, and optionally the extras you use to track which structures have been run.
+Be sure to check out the examples in the repository, and if you have any questions simply raise them on the [AiiDA Discourse](https://aiida.discourse.group/latest)!
+
+### [`aiida-hyperqueue`](https://github.com/aiidateam/aiida-hyperqueue/)
+
+If you've ever run simulations on a high-performance computing (HPC) infrastructure, you're likely familiar with scheduling systems such as SLURM, SGE and PBSPro (AiiDA supports all of these, by the way :wink:).
+However, you might run into issues when running many small calculations through AiiDA on these queueing systems.
+Maybe your HPC cluster has a low limit on the number of active jobs allowed at any time, or the total queueing time for your workflow becomes prohibitively large.
+In this case, you might want to have a look at the `aiida-hyperqueue` plugin, which interfaces AiiDA with the [HyperQueue scheduler](https://github.com/It4innovations/hyperqueue).
+This allows you to run many jobs in parallel on a single node, and avoids having to requeue for each calculation in your workflow.
+So if you ever need to run a lot of shorter simulations, make sure to give it a shot![^3]
+
+## AiiDA setup
+
+### [`aiida-project`](https://github.com/aiidateam/aiida-project/)
+
+As we outlined in [another one of our recent blog posts](https://aiida.net/news/posts/2024-09-20-simpler-installation.html), since v2.0, we have significantly simplified the creation of an AiiDA profile.
+Still, if you find yourself juggling many different profiles on your machine, each with its own configuration, Python environment, and directory of setup files (e.g., for `Computer` and `Code` instances), it can become easy to mess things up.[^4]
+To help out with this, `aiida-project` encapsulates and automates the steps to create and switch between different projects using AiiDA, basically functioning as an _AiiDA project manager_.
```

Comment:
```suggestion
To help out with this, `aiida-project` encapsulates and automates the steps to create and switch between different projects using AiiDA, basically functioning as an _AiiDA environment manager_.
```

I think _environment_ is more proper here, as it effectively sets up a Python environment.